### ISSUE TITLE
Need Help setting up basic email to admin

### ISSUE THREAD

**Issue by DavidWells** [inboundnow/leads#8]
Hey Hudson, I need your help getting the email part of this working for a basic basic v1 release of forms.

v1 will probably only have text and textarea inputs.

If you use: [inbound_form fields="First Name, Last Name, Email, Company, Phone" required="Company" textareas="Company"] it will generate a form.

[inbound_form] will make a single email field form.

The email function is here but haven't got it working https://github.com/inboundnow/leads/blob/master/shared/classes/form.class.php#L135

Basically the functionality goes like this: 
**on form submission send email to the WP admin email address by default unless they have the 'notify' parameter in shortcode with different email.**

So for email: [inbound_form fields="First Name, Last Name, Email" notify="tim@tim.com"] this form would email the form fields to tim not the admin =)

The current shortcode params are here but likely to change/grow/morph: https://github.com/inboundnow/leads/blob/master/shared/classes/form.class.php#L41
